doc/sphinx/source/recipes/recipe_kcs.rst

```diff
@@ -0,0 +1,141 @@
+    0  2050        P10     0.98  [2019, 2048]       0.99                    1.00
+    1  2050        P90     2.01  [2045, 2074]       2.02                    0.99
+    2  2085        P10     1.38  [2030, 2059]       1.38                    1.00
+    3  2085        P90     3.89  [2071, 2100]       3.28                    1.18
+
+
+which is printed to the log file and also saved as a csv-file ``scenarios.csv``.
+Additionally, a figure is created showing the CMIP spread in global temperature change,
+AND highlighting the selected steering parameters and resampling periods:
+
+.. _fig_kcs_global_matching:
+.. figure::  /recipes/figures/kcs/global_matching.png
+   :align:   center
+
+The diagnostic ``local_resampling`` procudes a number of output files:
+
+* ``season_means_<scenario>.nc``: intermediate results, containing the season means for each segment of the original target model ensemble.
+* ``top1000_<scenario>.csv``: intermediate results, containing the 1000 combinations that have been selected based on winter mean precipitation.
+* ``indices_<scenario>.csv``: showing the final set of resamples as a table:
+
+  .. code-block:: python
+
+                      control                                                      future
+                    Segment 0 Segment 1 Segment 2 Segment 3 Segment 4 Segment 5 Segment 0 Segment 1 Segment 2 Segment 3 Segment 4 Segment 5
+     Combination 0          5         7         6         3         1         3         2         4         2         4         7         7
+     Combination 1          0         3         0         4         3         2         4         1         6         1         3         0
+     Combination 2          2         4         3         7         4         2         5         4         6         6         4         2
+     Combination 3          1         4         7         2         3         6         5         3         1         7         4         1
+     Combination 4          5         7         6         3         1         3         2         3         0         6         1         7
+     Combination 5          7         2         1         4         5         1         6         0         4         2         3         3
+     Combination 6          7         2         2         0         6         6         5         2         1         5         4         2
+     Combination 7          6         3         2         1         6         1         2         1         0         2         1         3
+
+
+* Provenance information: bibtex, xml, and/or text files containing citation information are stored alongside the final result and the final figure.
+  The final combinations only derive from the target model data, whereas the figure also uses CMIP data.
+* A figure used to validate the final result, reproducing figures 5 and 6 from Lenderink et al.:
+
+.. _fig_kcs_local_validation:
+.. figure::  /recipes/figures/kcs/local_validation_2085.png
```

Comment:
WH temperature scenarios do not yet make sense in my version of the figure, should be at the top of the CMIP range.